src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,14 +4,17 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
@pranavkm open to suggestions, but cross-compiling will be messy. Consider this: the SDK includes net461 in PackageTargetFallback automatically when targeting .NET Core 2.0 and .NET Standard 2.0. I don't think this is a well-tested area, and it's possible we'll end up with the wrong assembly.